# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/clang/clang-tools-extra.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- clang-17.0.6.src/CMakeLists.txt.vanilla	2024-03-17 17:15:22.836572510 +0100
+++ clang-17.0.6.src/CMakeLists.txt	2024-03-17 17:15:51.201571081 +0100
@@ -468,6 +468,7 @@
 
 add_subdirectory(lib)
 add_subdirectory(tools)
+add_subdirectory(clang-tools-extra)
 add_subdirectory(runtime)
 
 option(CLANG_BUILD_EXAMPLES "Build CLANG example programs by default." OFF)
